packages/hop-node/src/config/validation.ts

```diff
@@ -68,32 +69,76 @@ export async function validateConfig (config?: FileConfig) {
     Chain.Polygon
   ]
 
+  const validTokenKeys = [
+    'USDC',
+    'USDT',
+    'DAI',
+    'ETH',
+    'MATIC',
+    'WBTC'
+  ]
+
+  let enabledChains: string[] = []
   if (config.chains) {
-    const networkKeys = Object.keys(config.chains)
-    await validateKeys(validNetworkKeys, networkKeys)
+    enabledChains = Object.keys(config.chains)
+    validateKeys(validNetworkKeys, enabledChains)
+    const validChainKeys = ['rpcUrl', 'waitConfirmations']
+    for (const chain in config.chains) {
```

Comment:
I believe we should to enforce that the RPC urls exist for all routes that the bonder has. For example, if the bonder is bonding on xDai -> Polygon, we should enforce that they have an RPC endpoint for xDai, Polygon, and mainnet (see comment below about mainnet).

Does this code already do that? If not, do you think this is practical/possible?

Reply:
Yes, it already validates that the chain config must exist for a route [here](https://github.com/hop-protocol/hop/blob/1878d754ab9e40806a9a1748530403686306e2bb/packages/hop-node/src/config/validation.ts#L172-L175)